lib/chat_web/live/main_live/page/dialog.ex

```diff
@@ -209,17 +209,6 @@ defmodule ChatWeb.MainLive.Page.Dialog do
     |> assign(:edit_message_id, nil)
   end
 
-  def delete_message(
```

Comment:
why to delete this? we have text message removal AFAIR

Reply:
I've searched for the reference. I don't think we do a single message deletion anywhere in the dialog and the room.